internal/commands/push/inputs_test.go

```diff
@@ -14,13 +15,27 @@ import (
 
 func TestPushInputsResolve(t *testing.T) {
 	t.Run("Should return an error if run from outside a project directory and no local flag is set", func(t *testing.T) {
-		profile, teardown := mock.NewProfileFromTmpDir(t, "app_init_input_test")
-		defer teardown()
+		profile := mock.NewProfile(t)
```

Comment:
I think this was the wrong place to make a change.... the `ProfileFromTmpDir` wasn't needed for the `"should return error when more than one dependencies flag is set"` test.  we should make this change there and revert it here

Reply:
Yeah..